src/sparsezoo/utils/numpy.py

```diff
@@ -104,13 +63,35 @@ def save_numpy(
     return export_path
 
 
+def _fix_loaded_numpy(array) -> Union[numpy.ndarray, Dict[str, numpy.ndarray]]:
+    if not isinstance(array, numpy.ndarray):
```

Comment:
Is this block assuming array is a `Dict`? should it checkout for isinstance(Dict)?